src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -627,6 +627,10 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
   private pasteDataValidation(origin: CellPosition, target: CellPosition) {
     const rule = this.getters.getValidationRuleForCell(origin);
     if (!rule) {
+      const targetRul = this.getters.getValidationRuleForCell(target);
+      if (targetRul) {
+        this.adaptDataValidationRule(target.sheetId, targetRul, [], [toXC(target.col, target.row)]);
```

Comment:
Nitpick: i'd add a comment that we're removing the DV rule on the target, kinda hard to see what we're doing by reading the code normally. adaptDataValidationRule are kinda obscure ^^'